vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -78,6 +78,7 @@
   <Import Project="..\Chakra\Chakra.vcxitems" Label="Shared" />
   <Import Project="..\JSI\Shared\JSI.Shared.vcxitems" Label="Shared" />
   <Import Project="..\Shared\Shared.vcxitems" Label="Shared" />
+  <Import Project="..\Mso\Mso.vcxitems" Label="Shared" />
```

Comment:
>Mso [](start = 22, length = 3)

(nit) would be nice if these were ordered :)

Reply:
VS will re-order them after sometime anyway. It does not seem to be worth the effort...

---
In reply to: [370870305](https://github.com/microsoft/react-native-windows/pull/3958#discussion_r370870305) [](ancestors = 370870305)